march_hardware/include/march_hardware/IMotionCube.h

```diff
@@ -64,6 +72,15 @@ class IMotionCube : public Slave
   bool goToOperationEnabled();
   bool resetIMotionCube();
 
+  void setActuationMode(ActuationMode actuationMode)
```

Comment:
why are these methods in the header file and not in .cpp

Reply:
Laziness probably, fixed